Pixie/Pixie/Classes/Layout/UIEdgeInsets/UIEdgeInsets+Dimensions.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UIEdgeInsets+Dimensions.swift
+//  Pixie
+//
+//  Created by Grzegorz Sagadyn on 19/04/2020.
+//
+
+import UIKit
+
+import UIKit
+
+extension UIEdgeInsets {
+    public var width: CGFloat {
```

Comment:
I was thinking about it as well - but I planned to name it as horizontal and vertical. This is just proposition you can also leave it as is

Reply:
What is funny, the first version of the above was horizontal and vertical. I'll change this as you suggested.